src/components/catalogtree/CatalogitemDirective.js

```diff
@@ -110,11 +110,20 @@ goog.require('ga_previewlayers_service');
                 });
 
               // Leaf
-              } else if (!gaBrowserSniffer.mobile) {
-                iEl.on('mouseenter', function(evt) {
+              } else {
+                var cancelMouseEvents = false;
+                iEl.on('touchstart mouseover', function(evt) {
+                  if (!gaEvent.isMouse(evt) || cancelMouseEvents) {
+                    cancelMouseEvents = true;
+                    return;
+                  }
                   addPreviewLayer(scope.map, scope.item);
-                }).on('mouseleave', function(evt) {
+                }).on('mouseout', function(evt) {
+                  if (!gaEvent.isMouse(evt)) {
+                    return;
+                  }
                   removePreviewLayer(scope.map);
+                  cancelMouseEvents = false;
```

Comment:
Similar code as above in the attribution directive. Could maybe be put into a service/common component.